hadoop-hdds/framework/src/test/java/org/apache/hadoop/hdds/security/x509/certificate/client/TestDefaultCertificateClient.java

```diff
@@ -75,7 +76,7 @@
   private Path dnMetaDirPath;
   private SecurityConfig omSecurityConfig;
   private SecurityConfig dnSecurityConfig;
-  private final static String UTF = "UTF-8";
+  private final static String UTF = StandardCharsets.UTF_8.name();
```

Comment:
This is used in two kinds of calls:

1. `RandomStringUtils.random`: its `chars` parameter is not intended for charsets, rather the set of characters to use.  Eg. for `"UTF-8"` it will generate strings like `"--8F8T8U8T"`.  Since we don't need this special kind of random string, this usage can be simply removed.
2. `IOUtils.toInputStream`: can accept `Charset`, too.

After removing the first kind of usage, we can convert this constant to `Charset`.  (I would also inline it, ie. simply use `UTF_8`.)

Reply:
done